pkg/build/admission/jenkinsbootstrapper/admission_test.go

```diff
@@ -0,0 +1,149 @@
+package jenkinsbootstrapper
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"k8s.io/kubernetes/pkg/admission"
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/auth/user"
+	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/fake"
+	"k8s.io/kubernetes/pkg/runtime"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
+	"github.com/openshift/origin/pkg/client/testclient"
+	configapi "github.com/openshift/origin/pkg/cmd/server/api"
+)
+
+func TestAdmission(t *testing.T) {
+	enableBuild := &buildapi.Build{Spec: buildapi.BuildSpec{CommonSpec: buildapi.CommonSpec{Strategy: buildapi.BuildStrategy{JenkinsPipelineStrategy: &buildapi.JenkinsPipelineBuildStrategy{}}}}}
+	testCases := []struct {
+		name           string
+		objects        []runtime.Object
+		jenkinsEnabled *bool
+
+		attributes  admission.Attributes
+		expectedErr string
+
+		validateClients func(kubeClient *fake.Clientset, originClient *testclient.Fake) string
+	}{
+		{
+			name:            "skip 1",
```

Comment:
would it make sense to name your test case by the reason it should be skipped ? (ie. "jenkins not enabled")